spec/rubocop/cop/rspec/rails/http_status_spec.rb

```diff
@@ -15,6 +15,17 @@
       RUBY
     end
 
+    it 'registers an offense when using string value' do
+      expect_offense(<<-RUBY)
+        it { is_expected.to have_http_status "200" }
+                                             ^^^^^ Prefer `:ok` over `"200"` to describe HTTP status code.
+      RUBY
+
+      expect_correction(<<-RUBY)
+        it { is_expected.to have_http_status :ok }
+      RUBY
+    end
```

Comment:
IDK if we should handle [this](https://github.com/rails/rails/blob/3668b4b5978822f1e6311c7fd7a32f58daee136a/actionpack/test/controller/render_test.rb#L950).

Reply:
Are you wondering whether to handle the following lines?

```ruby
    assert_equal 404, @response.response_code
    assert_equal "Not Found", @response.message
    assert_response :not_found
```

Reply:
No, status code strings with additional data like status: "404 Eat Dirt" there.